src/extra.ml

```diff
@@ -71,14 +71,21 @@ module Option =
       | (None    , None    ) -> true
       | (Some(e1), Some(e2)) -> eq e1 e2
       | (_       , _       ) -> false
+
+    (** [pp pp_elt oc o] prints on the channel [oc] the element [e] with
+       [pp_elt] if [o = Some e]. *)
+    let pp : 'a pp -> 'a option pp = fun pp_elt oc o ->
+      match o with
+      | None   -> ()
+      | Some e -> pp_elt oc e
```

Comment:
This seems to be unused. If yes, please, remove.

Reply:
Right but it could be useful later.

Reply:
If you really want to keep that function (and I think we should not maintain dead code), then please fix the indentation in the comment.